INSTALL.md

```diff
@@ -46,8 +46,13 @@ See [instructions](./docker/README.md)
 	go get -tags mongodb github.com/tinode/chat/server && go build -tags mongodb -o $GOPATH/bin/tinode github.com/tinode/chat/server
 	go get -tags mongodb github.com/tinode/chat/tinode-db && go build -tags mongodb -o $GOPATH/bin/init-db github.com/tinode/chat/tinode-db
 	```
+ - **All** (bundle all the above DB adapters):
```

Comment:
A couple of points:

1. A new tag is not really needed. We just need to add support for multiple tags like `go build -tags="mysql rethinkdb mongodb"`. This is more flexible than "all" and requires fewer code changes.

2. I think `all` is too generic. It's not namespaced to databases. It's just `all` for the entire project. Maybe `all_databases` and such.